src/applications/disability-benefits/all-claims/config/schema.js

```diff
@@ -34,6 +34,21 @@ const schema = {
           }
         }
       }
+    },
+    militaryRetiredPayBranch: {
+      type: 'string',
+      'enum': [
+        'Air Force',
+        'Army',
+        'Coast Guard',
+        'Marine Corps',
+        'National Oceanic and Atmospheric Administration',
```

Comment:
This is abbreviated in some of the other branch lists, does EVSS validate these branch values?

Reply:
Yes, they do, and they require a very strange format. Luckily, vets-api is doing this translation for us. In the Increase form, this is abbreviated, but I decided to just use the provided copy for this issue. I suppose one of these days we should probably decide whether we want to abbreviate it or spell it out across the board.